app/config/parameters.yml.dist

```diff
@@ -20,6 +20,13 @@ parameters:
     paypal.express_checkout.usd_testuser_login: EDIT ME
     paypal.express_checkout.usd_testuser_password: EDIT ME
 
+    # required only for authorization by merchant token
+    # see: https://developer.paypal.com/docs/classic/permissions-service/integration-guide/PermissionsAbout/
+    paypal.express_checkout.token: EDIT ME
+    paypal.express_checkout.tokenSecret: EDIT ME
+    ## e-mail of the merchant you work in behalf of
+    paypal.express_checkout.third_party_subject: EDIT ME
```

Comment:
if user needs a subject he can set it to the payment model directly, remote it from here and from the api options as well